exporting-grafana-snapshots.md

```diff
@@ -0,0 +1,48 @@
+---
+title: 导出 Grafana 监控数据为快照
+summary: 了解如何将 Grafana 监控数据导出为快照以及如何可视化快照文件。
+---
+
+# 导出 Grafana 监控数据为快照
+
+在故障诊断中，metrics 数据十分重要。当你请求远程协助时，技术支持人员有时需要查看 Grafana 仪表盘以确认问题所在。[MetricsTool](https://metricstool.pingcap.com/) 可以帮助将 Grafana 仪表盘的快照导出为本地文件，并将快照可视化。因此，你可以在不泄露 Grafana 服务器上其他敏感信息的前提下，将监控数据以快照形式分享给外部人员，同时也方便外部人员准确识读数据图表。
+
+## Usage 使用方法
+
+可以通过访问 <https://metricstool.pingcap.com/> 来使用 MetricsTool。它主要提供以下三种功能：
+
+* **导出快照**：提供一段在浏览器开发者工具上运行的用户脚本。可以使用这个脚本在任意 Grafana v6.x.x 服务器上下载当前仪表盘中所有可见面板的快照。
+
+    ![运行用户脚本后的 MetricsTool Exporter 截图](/media/metricstool-export.png)
+
+* **快照可视化**：提供一个快照导出文件网页端可视化工具。快照经过可视化后，操作起来与实际的 Grafana 仪表盘无异。
+
+    ![MetricsTool Visualizer 截图](/media/metricstool-visualize.png)
+
+* **导入快照**：介绍如何将导出的快照重新导入到已有的 Grafana 实例中。
+
+## FAQs
+
+### 与直接截图及导出 PDF 相比，MetricTool 有什么优势？
+
+MetricsTool 导出的快照文件包含快照生成时的 metrics 实际数值。你可以通过 Visualizer 与渲染的图表进行交互，比如切换 series、选择一个较小的时间范围以及检查特定时间点的 metrics 数据值等，就像在操作一个实际的 Grafana 仪表盘一样，因此它比 PDF 文件和截图更强大。
```

Comment:
```suggestion
MetricsTool 导出的快照文件包含快照生成时的监控指标实际数值。你可以通过 Visualizer 与渲染的图表进行交互，比如切换序列、选择一个较小的时间范围以及检查特定时间点的监控数据值等，就像在操作一个实际的 Grafana Dashboard 一样，因此它比 PDF 文件和截图更强大。
```

also unlike the English "docs", the word "Visualize" never appeared before this line, so it would be confusing for Chinese readers to understand what "Visualizer" is referring to.

Reply:
Add the tool name "Vizualizer" on L18.